### DISCUSSION TITLE
Vue Designer

### DISCUSSION THREAD

**Discussion by zfeihong** [elsa-workflows/elsa-core#5935] [Q&A]
I really like the design of Elsa; 
I've almost read all of your blog posts and design articles.

Can the Elsa framework be adapted for Vue? Our front-end development team uses Vue technology. 
So we want to replace the front-end editor with Vue.
I'd like to ask if there might be compatibility issues between the front-end and back-end, because I feel that the current editor is still quite tightly coupled with the back-end.

tks!

**Comment by sfmskywalker** [comment]
Thank you for the kind words! I'm glad to hear you’re enjoying the design of Elsa.

Yes, Elsa is designed to be UI-agnostic, so you can definitely create your own designer using Vue. Your custom designer can interact with the Elsa Server via the REST API, or you could even implement your own REST API if that better suits your needs.

Another option is to re-host the existing designer in your Vue app. Although the designer is implemented with Blazor, it can be wrapped as a custom element, which can then be integrated as a Vue component. There's a proof of concept for this using React here: [Elsa Studio Wrappers](https://github.com/elsa-workflows/elsa-studio/tree/main/src/wrappers).
